### PR TITLE
MaintenanceSolution fix

### PR DESCRIPTION
**Pull request by potatoqualitee**
Horrifyingly enough, `MaintenanceSolution` was listed as `MaintenancePlan` 

Also, all of the ola jobs have Ola prepended and that wasnt listed either

### CODE REVIEW

**Comment by shaneis** [comment]
Nice catch! I completely missed the prefix when I checked the first time. 👍

**Comment by potatoqualitee** [comment]
Ditto! Looking forward to integration tests. Just gotta find the time & energy.

**Comment by potatoqualitee** [comment]
It all works well, but I'm suspicious of this one

image

**Comment by shaneis** [comment]
Don't have Ola installed in work but I can do some testing when I get home tonight if you're still concerned.

**Comment by potatoqualitee** [comment]
just ran various tests locally - works. thank you for the sanity check, @shaneis !

File: checks/MaintenanceSolution.Tests.ps1 [hunk 1]
```diff
@@ -1,47 +1,47 @@
...
-    $Retention = Get-DbcConfigValue "policy.ola.$($JobPrefix)retention"     
-
+    $Scheduled = Get-DbcConfigValue "policy.ola.$($JobPrefix)scheduled"
+    $Retention = Get-DbcConfigValue "policy.ola.$($JobPrefix)retention"
+    
     #Write-PSFMessage -Level Host -Message "$jobname / $JobPrefix / $tagname"
-
-    Describe "Ola - $Jobname" -tags $tagname, OlaJobs, $filename {
+    
+    Describe "Ola - $Jobname" -Tag $tagname, OlaJobs, $filename {
```
Comment (shaneis) [inline checks/MaintenanceSolution.Tests.ps1]: hey @potatoqualitee looks like you changed plural to singular here...

**Comment by potatoqualitee** [comment]
ah dang,t hank you. @SQLDBAWithABeard can we get unit tests on branches?

File: checks/MaintenanceSolution.Tests.ps1 [hunk 2]
```diff
@@ -1,47 +1,47 @@
 $filename = $MyInvocation.MyCommand.Name.Replace(".Tests.ps1", "")
 
-Describe "Ola maintenance solution installed" -Tags OlaInstalled, $filename{
+Describe "Ola maintenance solution installed" -Tag OlaInstalled, $filename{
```
Comment (shaneis) [inline checks/MaintenanceSolution.Tests.ps1]: And singular here as well...

**Pull request merged**